environment.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
 - conda-forge::scikit-learn=0.21.3
 - conda-forge::black=19.10b0
 - conda-forge::seaborn=0.9.0
+- conda-forge::pillow=4.1.2
```

Comment:
I got a conda install error for this version of pillow.

```sh
Solving environment: failed

ResolvePackageNotFound: 
  - conda-forge::pillow=4.1.2
```

Adding [pillow=6.2.1](https://anaconda.org/conda-forge/pillow/files) seems to have worked.

Will this break the pipeline?

Reply:
Added in de28f65450f49b917de78451b4eda8226e944edb. Still worth noting if the upgraded version breaks anything :)

Reply:
Thanks completely overlooked it